cairo-sys-rs/src/lib.rs

```diff
@@ -21,6 +21,7 @@ pub mod enums;
 
 use enums::{
     Status,
+    Content,
```

Comment:
Oh, I put "Content" instead of "cairo_content_t" because that is the pattern used for Status/cairo_status_t already.

I see now that there is an empty

```
#[repr(C)]
pub struct cairo_content_t(c_void);

```

I'll remove that.

So would you prefer if I used "pub type cairo_content_t = enums::Content" and left all the function prototypes with cairo_content_t?